tirgei/Questioner
# #162979132 A user can view available meetups

#### What does this PR do?
This Pull Request creates the main page where users can view meetups

#### Description of Task to be completed?
The tasks for this PR is:
- Create the main page 
- Style the main page
- Add the all meetups page
- Style the all meetups page

#### How should this be manually tested?
Clone this repository on your machine:
```$ git clone https://github.com/tirgei/Questioner.git ```

CD into the project folder on your machine
```$ cd Questioner ```

Check out the branch for this PR
``` $ git checkout ft-user-meetups-162979132```

Open the `index.html` file on your browser

Alternatively, you can view the site here: [Questioner](https://tirgei.github.io/Questioner/UI)


#### Pivotal Tracker story associated with this PR
[#162979132](https://www.pivotaltracker.com/story/show/162979132)

#### Screenshots
![index1](https://user-images.githubusercontent.com/17829576/50724009-0e09ca80-10f7-11e9-8fc7-23ef0db1f4a2.png)

![index2](https://user-images.githubusercontent.com/17829576/50724010-1235e800-10f7-11e9-8d4e-332f241f3149.png)

![index3](https://user-images.githubusercontent.com/17829576/50724014-15c96f00-10f7-11e9-8ec2-2f265b66853a.png)

![index4](https://user-images.githubusercontent.com/17829576/50724018-182bc900-10f7-11e9-95d1-052b962f6f7f.png)

![meetups](https://user-images.githubusercontent.com/17829576/50724021-1d891380-10f7-11e9-8181-a2e62627daa0.png)


Mobile 
![mobile1](https://user-images.githubusercontent.com/17829576/50724022-2548b800-10f7-11e9-8faf-75352381737b.png)

![mobile2](https://user-images.githubusercontent.com/17829576/50724023-2974d580-10f7-11e9-8d03-849ea43d2881.png)

![mobile3](https://user-images.githubusercontent.com/17829576/50724024-2aa60280-10f7-11e9-8297-cccb9194b735.png)
